web/src/status.tsx

```diff
@@ -1,11 +1,12 @@
 import { isZeroTime } from "./time"
+import { RuntimeStatus } from "./types"
 
 // A combination of runtime status and build status over a resource view.
 // 1) If there's a current or pending build, this is "pending".
 // 2) Otherwise, if there's a build error or runtime error, this is "error".
 // 3) Otherwise, we fallback to runtime status.
-function combinedStatus(res: any): string {
-  let runtimeStatus = res.RuntimeStatus
+function combinedStatus(res: any): RuntimeStatus {
```

Comment:
...i'm confused about whether runtime status and build status are two separate things or not. this comment indicates that this func returns some combination of build status and runtime status, but returns a `RuntimeStatus`, and i'm confused.

Reply:
Yeah let me rename this.